modules/datastore/tests/src/Unit/Controller/QueryDownloadControllerTest.php

```diff
@@ -346,7 +347,7 @@ public function testStreamedBadSchema() {
    * @return \MockChain\Chain
    *   MockChain chain object.
    */
-  private function getQueryContainer(int $rowLimit) {
+  private function getQueryContainer(int $rowLimit, ?int $responseStreamMaxAge = NULL) {
```

Comment:
This method needs a docblock update, but not because of your changes. :-)

It needs two `@params` and also `@returns` a `MockObject` and not a `Chain`.

Reply:
Docblock now looks like this: 

/**
   * Create a mock object for the main container passed to the controller.
   *
   * @param int $rowLimit
   *    The row limit for a query.
   * @param int|null $responseStreamMaxAge
   *    The max age for the response stream in cache, or NULL to use the default.
   * @param array $info
   *   Dataset info array mock to be returned by DatasetInfo::gather().
   *
   * @return \PHPUnit\Framework\MockObject\MockObject
   *   MockChain mock object.
   */

Reply:
Thanks, but there's no `$info` parameter.

Reply:
That was there before, I didn't add it. Should I remove it?